modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/entity/ModelType.java

```diff
@@ -30,10 +30,13 @@
 @Data
 @Entity
 public class ModelType {
-    
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
-    
-    public String key;
+
+    private String key;
+
+    private boolean seyrenFlag;
```

Comment:
1. This belongs on the model table, not the model_type table. See below.
2. Let's not expose this on the entity itself. This is a purely internal flag for now and we don't want people to start developing to it.